DataFormats/QualityControl/src/TimeRangeFlagCollection.cxx

```diff
@@ -0,0 +1,78 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+// O2 include
+#include "DataFormatsQualityControl/TimeRangeFlagCollection.h"
+
+#include <iostream>
+
+ClassImp(o2::quality_control::TimeRangeFlagCollection);
```

Comment:
this is not really useful any more as it was used to generate the THTML for the code documentation. We use doxygen now.